lib/cloudshaper/secrets.rb

```diff
@@ -1,29 +1,18 @@
 require 'json'
 require 'open3'
 
-# Load and provide access to secrets required by terraform providers
-class SecretHash < Hash
-  class SecretNotFound < Exception; end
+SECRETS ||= {}
```

Comment:
I still think this should be `Cloudshaper::SECRETS` so we don't pollute the global namespace.